### ISSUE TITLE
Tip4Commit Issue

### ISSUE THREAD

**Issue by super3** [super3/peercoin_website_old#223]
Although I love that people are donating, I think we should consider removing some of the funds for the tip4commit. Don't really want to reward someone $100 for fixing a comma. Those funds could be put to better use.

**Comment by sashazykov** [comment]
Do you think that we should limit tip size? I thought so, but @arsenische does not agree. He thinks that donors should decide this by their donations.

**Comment by danbartram** [comment]
I totally agree. It would be better for the repo owner, or a scale of "usefulness" (e.g. 1% to 100% of the expected tip) or value ("minor", "major", etc.) to decide how much each commit gets tipped.

Would it be possible to get Tip4Commit to scan the comments in the pull request, find comments from the repo owner and look for something with the syntax:

```
tip4commit: minor
```

Then that would be parsed into a percentage split of the "expected tip" (1%), so for example minor would be 15% maybe.

That would tip the minor contributor 15% of the 1% expected tip.

At a 0.1 BTC expected tip, this would tip them 0.015 bitcoin instead.

```
tip4commit: 0.001 BTC
```

Could be another approach to this too, but obviously you'd need to make sure the system waited for that tip, and not process the full 1% automatically.

What should happen if the repo owner forgets to comment on the pull request?

**Comment by arsenische** [comment]
Limiting the tip amount could be useful, but I am afraid that it could diminish the incentive for new developers to join the projects. If donors are not satisfied with the pace of development, they can donate more increasing the tip size and attracting more attention to projects. Demand and supply should eventually set the reasonable tip size.

AFAIK there is ongoing work on https://github.com/tip4commit/tip4commit/issues/11 that should help (fixing a comma could be a #tiny or even #freebie commit). Project maintainer would be able to evaluate commits if they want, but they probably shouldn't worry too much about donors' money.

**Comment by super3** [comment]
I like the ideas discussed, but would also like to be able to put tips on specific pull requests to guide development. I'm going to do a separate fund for development. I'd rather have guided development, and then have tip4commit handle the small fixes.

**Comment by malaga2333** [comment]
create a sequence of files (more than 200), and push them as commits, you'll get the whole balance of the project and use it for guided development :)) 

any thanks accepted
btc 19Wa6kXdeA2XZqemCzSrd5ExFyk6BBAjZp
ppc PCgx2WF8fdQhTTeDYUUEVM4tNnWyeYBTUS
